PD-1674 / 25.04 / PD-1674-custom-app-ui-ref-update (by linzi-ix) (#3564)
truenas/documentation: static/includes/apps/CustomAppIntro.md

To deploy a custom application, go to **Apps** and click **Discover Apps**.
Click **Custom App** to open the **[Install iX App]({{< relref "installcustomappscreens.md #install-ix-app-screen" >}})** screen with a guided installation wizard.
Click <i class="material-icons" aria-hidden="true" title="more_vert">more_vert</i> > **Install via YAML** to open the **[Custom App]({{< relref "installcustomappscreens.md #custom-app-screen" >}})** screen with an advanced YAML editor for deploying apps using Docker Compose.

TrueNAS also allows users to convert already installed apps into custom applications. 
Click on the app to wish to convert, and then click the widget next to **Edit** to open the menu containing **Update** and **Convert to custom app** options.

{{< trueimage src="/images/SCALE/Apps/ConvertToCustomAppMenu.png" alt="Convert to Custom App Menu" id="Convert to Custom App Menu" >}}

Select **Convert to custom app** from the dropdown list, then select **Confirm** to convert the app. 

{{< trueimage src="/images/SCALE/Apps/ConvertToCustomAppConfirm.png" alt="Convert to Custom App Confirm" id="Convert to Custom App Confirm" >}}

> Once converted, editing the information of a custom app opens an **Edit App YAML** window that gives users the option to enter a custom app configuration.

Reviewer: "After" or "when", but not "once", also I'm having trouble parsing this sentence. When does "Edit App YAML" window open? And where do you edit the information? is it "Edit the information in the xyz window or YAML file, or????,  what are you editing before the Edit App YAML window? Please reword this sentence so it is more clear what and where the user does whatever it is they do.

Reviewer 2: fixed--didn't parse over my changes from the cherry pick the first time

> After converting, clicking the **Edit** button within the **Application Info** window opens an **Edit App YAML** window. This gives users the option to enter a custom app configuration.

Reviewer: Better, but really wordy. How about something like this?
```suggestion
After converting, click **Edit** button within the **Application Info** window to open the **Edit App YAML** window and enter your a custom app configuration.
```

Reviewer 2: slightly modified to fix typos
